tools/zod2md-jsdocs/docs/zod2md-jsdocs-nx-plugin.md

```diff
@@ -0,0 +1,122 @@
+# @code-pushup/zod2md-jsdocs-nx-plugin
+
+The Nx Plugin for [zod2md](https://github.com/matejchalk/zod2md), a tool for generating documentation from Zod schemas.
+
+Why should you use this plugin?
+
+- Zero setup cost. Just add a `zod2md.config.ts` file and you're good to go.
+- Automatic target generation
+- Minimal configuration
+- Automated caching and dependency tracking
+
+## Usage
+
+```jsonc
+// nx.json
+{
+  //...
+  "plugins": ["./tools/zod2md-jsdocs-nx-plugin/src/lib/plugin.js"],
```

Comment:
The path `./tools/zod2md-jsdocs-nx-plugin/src/lib/plugin.js` is incorrect; it's actually `./tools/zod2md-jsdocs/dist/src` (as used in `nx.json`). It appears several times in this document.